src/engraving/libmscore/chord.cpp

```diff
@@ -3749,9 +3749,17 @@ void Chord::layoutArticulations()
 
         if (bottom) {
             if (!headSide && stem()) {
-                y = upPos() + stem()->length();
-                if (beam()) {
-                    y += score()->styleS(Sid::beamWidth).val() * _spatium * .5;
+                auto userLen = stem()->userLength();
+                if (_up) {
+                    y = downPos() - stem()->length() - userLen;
+                    if (beam()) {
+                        y -= score()->styleS(Sid::beamWidth).val() * _spatium * .5;
```

Comment:
This line

score()->styleS(Sid::beamWidth).val() * _spatium * .5;

is present in 4 places. It is better to add a temporary variable for it and reuse the variable 4 times

Reply:
Okay!